src/qagame/qagame.py

```diff
@@ -23,4 +24,5 @@
 	qvm.add_c_code(f.read(), ["../vanilla_29_sdk/cgame", "../vanilla_29_sdk/game"])
 qvm.replace_calls("ClientCommand", "ClientCommand_Hook")
 qvm.replace_calls("fire_grapple", "fire_grapple_Hook")
+qvm.replace_calls("G_Say", "G_Say_Hook")

```

Comment:
It might be nice to automagically replace all *_Hook calls at some point, though I guess that could be a little dangerous.

Reply:
Maybe as part of #11